cli/src/maintenance.rs

```diff
@@ -60,8 +63,11 @@ pub enum MaintenanceOutput {
         /// This is only an expected value, because a different transaction might
         /// execute between us observing the state and concluding that there is
         /// a difference, and our `WithdrawInactiveStake` instruction executing.
-        #[serde(rename = "expected_difference_lamports")]
-        expected_difference: Lamports,
+        #[serde(rename = "expected_difference_stake_lamports")]
+        expected_difference_stake: Lamports,
+
+        #[serde(rename = "unstaked_amount_lamports")]
+        unstaked_amount: Lamports,
```

Comment:
Huh, why does `unstaked_amount` go in the `WithdrawInactiveStake` output? `WithdrawInactiveStake` does not itself unstake anything? Is this the amount that we expect to withdraw from unstake accounts to the reserve?

Maybe call it `unstake_withdrawn_to_reserve` or something like that?

Reply:
Yeah, that was copy-pasta bug, fixed it